docs/blog/posts/2018-10-25-robolectric-4-0.md

```diff
@@ -7,6 +7,7 @@ authors:
 slug: robolectric-4-0
 ---
 
+<!-- markdownlint-disable-next-line MD026 -->
```

Comment:
The default is to not use trailing punctuation in titles. Here I think it makes sense to keep it.

https://github.com/DavidAnson/markdownlint/blob/v0.36.0/doc/md026.md